### PR TITLE
JIRA-OSDOCS3173: Updated AWS firewall allowlist

### PR DESCRIPTION
**Pull request by sayjadha**
**This PR is to address the JIRA issue**: https://issues.redhat.com/browse/OSDOCS-3173

**Updated topic**:
https://deploy-preview-44546--osdocs.netlify.app/openshift-rosa/latest/rosa_planning/rosa-aws-prereqs#osd-aws-privatelink-firewall-prerequisites

**Following are the updates**:

1. Added a row to the second table in step 3 at the [AWS PrivateLink firewall prerequisites](https://docs.openshift.com/rosa/rosa_getting_started/rosa-aws-prereqs.html#osd-aws-privatelink-firewall-prerequisites) containing:
URL: servicequotas.<aws region>.amazonaws.com 
Function: Required. Used to confirm quotas for deploying the service.
Port: 443, 80

2. Added another row to the same table containing:
URL: tagging.<region>.amazonaws.com
Function: Allows the assignment of metadata about AWS resources in the form of tags.
Port: 443, 80

3. Renamed the title here from "AWS PrivateLink firewall prerequisites" to "AWS firewall prerequisites".

4. Added a note stating that the table in step 1 on that page stating that "Creating a firewall with a ROSA private cluster (non-PrivateLink) is not supported."

**Repo**: Request cherrypick to enterprise-4.10 and enterprise-4.11

### CODE REVIEW

**Comment by xueli181114** [comment]
LGTM

File: modules/osd-aws-privatelink-firewall-prerequisites.adoc [hunk 1]
```diff
@@ -73,6 +73,9 @@ This section provides the necessary details that enable you to control egress tr
 |Provides core container images as a fallback when quay.io is not available.
 |===
 +
+[NOTE]
```
Comment (ahardin-rh) [inline modules/osd-aws-privatelink-firewall-prerequisites.adoc:76]: This admonition should include the ==== format as seen in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions

**Comment by ahardin-rh** [comment]
Just one comment from me. Thanks!

**Comment by sayjadha** [comment]
Hi peer review squad, I incorporated @ahardin-rh's feedback but for some reason the Travis build didn't run after I committed the changes. Could you please verify the change and merge the PR? Thank you.

**Review by bergerhoffer — COMMENTED** [review]
LGTM too!

**Pull request merged**